rest/route/user.go

```diff
@@ -159,6 +159,9 @@ func (h *userPermissionsPostHandler) Run(ctx context.Context) gimlet.Responder {
 	if !valid {
 		return gimlet.NewTextInternalErrorResponse("unexpected type of user found")
 	}
+	if dbuser == nil {
+		return gimlet.NewTextInternalErrorResponse("error converting gimlet user to DBUser")
```

Comment:
Can you change the error to match the one at 148 closer? Really it is the same error, it's just that interfaces can be not nil while the underlying structure _is_ actually nil (#justgothings) https://tour.golang.org/methods/12